servicecontrol/logging.md

```diff
@@ -10,18 +10,71 @@ redirects:
 
 ### Logging
 
-Instances of the ServiceControl service write logging information and failed message imports to the file system.  
-Before v1.9 the logging level by default was set to `Info`, however this level can be quite verbose, so from v1.9 the default logging level is now `Warn`, this level is also configurable by adding the following to the configuration file:
+Instances of the ServiceControl service write logging information and failed message import stack traces to the file system.  
+
+
+#### Version 1.8.3 and below 
+
+The default logging level was `Info`, 
+
+#### Version 1.9 and above
+
+The default logging level is `Warn`, this level is now configurable by adding the following to the `appSettings` section of the  configuration file:
 
 ```xml
 <!-- Log Level Options: Trace, Debug, Info, Warn, Error, Fatal, Off -->
 <add key="ServiceControl/LogLevel" value="Info" /> 
 ```
 
-NOTE: If ServiceControl experiences a critical exception when running as a service the exception information will be logged to the Windows EventLog rather than the log file.
+### RavenDB Logging 
```

Comment:
Is it worth having a sentence in here about why we have a Raven Logging? Something like:

> ServiceControl stores data in an embedded RavenDB database which generates it's own log messages.